src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/GerritJcascConfigurator.java

```diff
@@ -91,7 +90,12 @@ public Class<PluginImpl> getTarget() {
     protected void configure(
         Mapping config, PluginImpl instance, boolean dryrun, ConfigurationContext context
     ) throws ConfiguratorException {
-        List<GerritServer> oldServers = new ArrayList<>(instance.getServers());
+        if (!dryrun) {
+            for (GerritServer oldServer : instance.getServers()) {
+                oldServer.stopConnection();
```

Comment:
But if `super.configure(config, instance, dryrun, context);` now fails you will be left with the old configuration but stopped connections.